src/plugins/xfconf/xfconf.c

```diff
@@ -0,0 +1,206 @@
+/**
+ * @file
+ *
+ * @brief Source for xfconf plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include "xfconf.h"
+
+#include <kdb.h>
+#include <kdbease.h>
+#include <kdbhelper.h>
+#include <kdblogger.h>
+#include <xfconf/xfconf.h>
+
+int elektraXfconfOpen (Plugin * handle ELEKTRA_UNUSED, Key * errorKey ELEKTRA_UNUSED)
+{
+	ELEKTRA_LOG ("try to initialize xfconf\n");
+	GError * err = NULL;
+	if (xfconf_init (&err))
+	{
+		ELEKTRA_LOG_DEBUG ("succeed initielize xfconf\n");
+		return ELEKTRA_PLUGIN_STATUS_SUCCESS;
+	}
+	else
+	{
+		ELEKTRA_LOG ("unable to initialize xfconf(%d): %s\n", err->code, err->message);
+		g_error_free (err);
+		return ELEKTRA_PLUGIN_STATUS_ERROR;
```

Comment:
You should always call one of the `ELEKTRA_SET_*_ERROR` if you return `ELEKTRA_PLUGIN_STATUS_ERROR`.